defaults/main.yml

```diff
@@ -7,9 +7,10 @@ otp_jar_suffix: "-shaded"
 otp_jar_sha1: "7719d90f4b33bd877b85fb63db66820637b5612f"
 otp_process_mem: 3G
 otp_web_port: 8080
-otp_upstart_start_on: "(local-filesystems and net-device-up IFACE!=lo)"
+otp_service_after: 'network-online.target'
```

Comment:
Interesting. I was unaware of the existence of `network-online`. Almost all examples I've seen just use `network`, but the distinction of having the network be "up" seems valuable.

Reply:
This came from an old PR from @KlaasH that added support for `systemd`, but predated the ansible support.